portfolio/src/main/webapp/style.css

```diff
@@ -12,6 +12,16 @@
     background-size: contain;
 }
 
+button {
+    font-family: "Lato", 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    color: #2E4A62;
+    font-size: 16px;
+}
```

Comment:
Thank you for catching this! I did change it to specify the fetch button for one of the commits (it's not there anymore since I chose to get rid of the button and directly display the comments)